ballerina-tests/tests/test_commons.bal

```diff
@@ -29,7 +29,7 @@ const string TRANSFER_ENCODING = "transfer-encoding";
 const string LOCATION = "location";
 const string ORIGIN = "origin";
 const string ALLOW = "Allow";
-const string LINK = "Link";
+const string LINK = "link";
```

Comment:
@Dil-G this is the test constant. We need to find the same constant in the source dir and rename it